.github/renovate.json

```diff
@@ -1,20 +1,27 @@
 {
-  "schedule": ["every weekend"],
-  "labels": ["dependencies"],
+  "extends": [ "config:base", ":dependencyDashboardApproval" ],
+  "constraints": {
+    "node": "< 16"
+  },
+  "labels": [ "dependencies" ],
   "separateMajorMinor": "false",
```

Comment:
If we are using the dashboard issue informationally, I would be curious to see what it comes up with for Major bump suggestions as well as Minor bump suggestions -- would instantly be a clue that there's more than one path forward on upgrading a given dependency.

So, we could delete this line?